src/components/common/ConnectWallet/ConnectionCenter.tsx

```diff
@@ -10,9 +10,16 @@ import PairingDetails from '@/components/common/PairingDetails'
 import css from '@/components/common/ConnectWallet/styles.module.css'
 import { useCurrentChain } from '@/hooks/useChains'
 import { isPairingSupported } from '@/services/pairing/utils'
+import AppStoreButton from '@/components/common/AppStoreButton'
+import { isAppleMobileDevice } from '@/utils/detectMobileDevices'
 
 const ConnectionCenter = (): ReactElement => {
   const chain = useCurrentChain()
+  const [showDownload, setShowDownload] = useState(false)
+
+  setTimeout(() => {
+    setShowDownload(isAppleMobileDevice())
+  })
```

Comment:
`navigator` was not defined in the first render so I couldn't detect the device based on `navigator.userAgent`